lib/ReactViews/ExplorerWindow/Tabs.jsx

```diff
@@ -36,6 +36,10 @@ const Tabs = observer(
       this.props.viewState.myDataIsUploadView = false;
     },
 
+    async onUrlAddFinished() {
+      await this.props.viewState.viewCatalogMember();
```

Comment:
Although this call doesn't err out, I think the expectation is we pass a catalog member to [viewCatalogMember](https://github.com/TerriaJS/terriajs/blob/c465cf1288316f5b82043db8e9f33f634c31eb87/lib/ReactViewModels/ViewState.ts#L590) as a parameter and the method will focus the catalog member in the explorer view. 

What we could do instead is call [openAddData()](https://github.com/TerriaJS/terriajs/blob/main/lib/ReactViewModels/ViewState.ts#L540) which will just open the Data Catalogue.

Reply:
updated, this is a nicer solution